charts/orchestrator-infra/templates/serverless/knatives.yaml

```diff
@@ -6,6 +6,10 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: knative-serving
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation
```

Comment:
> Error: UPGRADE FAILED: pre-upgrade hooks failed: warning: Hook pre-upgrade orchestrator-infra/templates/serverless/knatives.yaml failed: object is being deleted: namespaces "knative-eventing" already exists

From the [CI error](https://github.com/redhat-developer/rhdh-chart/actions/runs/14220129811/job/39845845847?pr=117#step:13:133), it looks like Helm is trying to delete the namespace when attempting to upgrade the release. And namespace deletion is not fully synchronous.
Would it make sense to not delete the namespace once created, and leave it as it is? I think even just a pre-install hook would probably be enough for now..
So something like this:
```
annotations:
    "helm.sh/hook": pre-install
    "helm.sh/hook-weight": "-5"
    "helm.sh/resource-policy": keep
```

Reply:
Let's merge this and unblock the other PR. Thanks.